lms/static/sass/course/_info.scss

```diff
@@ -31,8 +57,9 @@ div.info-wrapper {
         }
 
         h2 {
-          font-size: $body-font-size;
-          font-weight: bold;
+          margin-bottom: ($baseline/4);
+          text-transform: none;
+          font-size: 10px;
```

Comment:
use $extends from _typography.scss where possible for this kind of change. thanks!